### ISSUE TITLE
Feature Request: Arrange Home screen apps Manually 

### ISSUE THREAD

**Issue by HalfBloodprinc3** [MM2-0/Kvaesitso#514]
home screen apps are arranged by their use and they are getting rearranged whenever I use any app more. So add an option to arrange the home screen app manually.

**Comment by exstntlmsnthrp** [comment]
Are you referring to the favourites on your home screen?

If yes, you are already able to do this. 

Select the pencil icon in the favourites bar, then you are able to drag and drop into your preferred order. 

![Screenshot_kvaes.png](https://github.com/MM2-0/Kvaesitso/assets/57594290/05518884-b2b9-4fbb-9b19-efa6fc2fe731)

If I've misunderstood, please explain further.

**Comment by HalfBloodprinc3** [comment]
> Are you referring to the favourites on your home screen?
> 
> If yes, you are already able to do this. 
> 
> Select the pencil icon in the favourites bar, then you are able to drag and drop into your preferred order. 
> 
> ![Screenshot_kvaes.png](https://github.com/MM2-0/Kvaesitso/assets/57594290/05518884-b2b9-4fbb-9b19-efa6fc2fe731)
> 
> If I've misunderstood, please explain further. 

No. I tried the favourite app function. But the apps are getting rearranged based on their use. I simply want a feature so that the pin apps remain fixed.

**Comment by exstntlmsnthrp** [comment]
Interesting, because they remain arranged in the order I set them.

**Comment by MM2-0** [comment]
Make sure to drag them to the "manually sorted" section of the favorites edit sheet. Otherwise they will be sorted based on usage.

**Comment by ElijahHW** [comment]
I was wondering about the same, but was not aware of the "manual" mode, thanks @MM2-0

**Comment by AbsurdlySuspicious** [comment]
Is it possible to also rearrange apps in tags? It would be really useful if you could arrange apps simlar to manual pinning in favorites.

**Comment by jay-knight** [comment]
I also would like to manually order apps in tags, and the order of the tags. I tend to rely on the a muscle/position memory when finding apps, and would like to order things in a way that make the most sense to me.

**Comment by exstntlmsnthrp** [comment]
You can already manually order tags. Tap the pencil icon, then press and hold on a tag, next drag it to your desired order. 

Unfortunately, there is no way yet (that I'm aware of) to manually sort apps within a given tag. 

I would also like to see the ability to manually sort within tags.
